pkg/controller/autoscaling/elasticsearch/controller.go

```diff
@@ -42,7 +42,9 @@ const (
 	enterpriseFeaturesDisabledMsg = "Autoscaling is an enterprise feature. Enterprise features are disabled"
 )
 
-var defaultRequeue = reconcile.Result{
+// licenseCheckRequeue is the default duration used to retry a licence check if the cluster is supposed to be managed by
+// the autoscaling controller and if the licence is not valid.
+var licenseCheckRequeue = reconcile.Result{
```

Comment:
What is the motivation for using a different interval on invalid licenses? To avoid a bad experience for users that set `syncPeriod` to a really high value? Initially I thought it would be to protect the operator from users that set it to a really low value because the license check is relatively expensive. But I realise that the check runs on every reconciliation loop anyway. Maybe worth a comment.

Reply:
> To avoid a bad experience for users that set syncPeriod to a really high value?

Yes exactly, my initial thought was that if the license is invalid it might be a user mistake, and we may want to trigger a new loop sooner than later, regardless of the `syncPeriod` value which serves an other purpose.